parser.js

```diff
@@ -76,7 +76,7 @@ var cssOptions = function () {
     type: 'css',
     prop_url: 'styleUrls',
     prop: 'styles',
-    pattern: /styleUrls\s*:\s*\[([^\]]*)\]/g
+    pattern: /styleUrls\s*:\s*\[([^\]]*)]/g
```

Comment:
Why did you unescape this bracket ?

Reply:
Ok, sorry, this line change wasn't indended to be pushed. Anyway, the closing square bracket should not be escaped in this regex: https://stackoverflow.com/a/43044993 (yeah, the question is about Java regex but it's also true for JS)

Reply:
ok, will merge soon. thanks